ducktape/tests/loader.py

```diff
@@ -154,21 +155,23 @@ def import_modules(self, file_list):
 
             # Try all possible module imports for given file
             path_pieces = f[:-3].split("/")  # Strip off '.py' before splitting
+            success = False
             while len(path_pieces) > 0:
                 module_name = '.'.join(path_pieces)
                 # Try to import the current file as a module
                 try:
                     module_list.append(importlib.import_module(module_name))
                     self.logger.debug("Successfully imported " + module_name)
+                    success = True
                     break  # no need to keep trying
                 except Exception as e:
-                    self.logger.debug("Could not import " + module_name + ": " + e.message)
-                    self.logger.debug(
-                        "   But it's ok: at most one import should work per module, so import failures are expected.")
                     continue
                 finally:
                     path_pieces = path_pieces[1:]
 
+            if not success:
+                self.logger.debug("Unable to import %s" % f)
```

Comment:
Sorry, I should have been clearer. Cleaning up the logs so it doesn't spit out about 80 debug messages for a normal, pretty small set of tests is good. I think that change makes sense.

I was just pointing out that this terse message may not give enough info. But now that I think about it more, the only way it can be confusing is if the set of files we're filtering to isn't restrictive enough, but since we filter down to `.py` files, I think this is fine.